Ktisis/Locale/i18n/fr_FR.json

```
@@ -0,0 +1,407 @@
+{
+	"$meta": {
+		"displayName": "French",
+		"selfName": "Français",
+		"__comment": "Using null element to signal 'et al' in the array below. Don't do this for your own localization.",
+		"maintainers": [
+			"@Luna664",
+			null
+		]
+	},
+
+	"tab": {
+		"general": {
+			"title": "Général"
+		},
+		"settings": {
+			"title": "Paramètres"
+		}
+	},
+
+	"config": {
+
+		"title": "Paramètres de Ktisis",
+
+		"interface": {
+			"title": "Interface",
+			"general": {
+				"title": "Général",
+				"openMethod": {
+					"fieldLabel": "Ouvrir Ktisis",
+					"Manually": "manuellement via `/ktisis`",
+					"OnPluginLoad": "lorsque le plugin se lance",
+					"OnEnterGpose": "lors de la pose en groupe"
+				},
+				"hideCharName": "Cacher le nom du personnage",
+				"censorNSFW": "Cacher le contenu explicite"
+			},
+			"transformTable": {
+				"title": "Table des transformations",
+				"digitPrecision": "Précision des chiffres",
+				"basePositionSpeed": "Vitesse de base (Position)",
+				"baseRotationSpeed": "Vitesse de base (Rotation)",
+				"baseScaleSpeed": "Vitesse de base (Mise à l'échelle)",
+				"ctrlSpeedMult": "Ctrl multiplie la vitesse",
+				"shiftSpeedMult": "Maj multiplie la vitesse",
+				"showSpeedMultiplierInputs": "Afficher les touches du multiplicateur de vitesse"
+			}
+		},
+
+		"overlay": {
+			"title": "Superposition",
+
+			"skeleton": {
+				"header":"Lignes et points du squelette",
+				"drawLines": "Dessiner des lignes sur le squelette",
+				"__comment": "NOTE : Ceux-la devraient refléter plus clairement la fonctionnalité réelle (dessiner des lignes/points lorsque le *gizmo est actif*)."",
```

JSON syntax error here.
```suggestion
				"__comment": "NOTE : Ceux-la devraient refléter plus clairement la fonctionnalité réelle (dessiner des lignes/points lorsque le *gizmo est actif*).",
```